pkgs/applications/misc/hyperspace/default.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv, fetchurl, lib
```

Comment:
Please only input stdenv or lib.

Reply:
Thanks.  In light of recent RFC or whatever on not using stdenv.lib I think this is preferred now so will leave as-is.